k8s/base/nginx.tmpl

```diff
@@ -0,0 +1,1609 @@
+{{ $all := . }}
```

Comment:
I needed to tweak the default `nginx` configuration to allow SSL termination of a TCP service into a different port and act as a proxy. This is done by mounting this template file into the `nginx` POD. This solution came from https://github.com/kubernetes/ingress-nginx/issues/636#issuecomment-749026036.